static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -108,7 +113,12 @@ export class ProteinProteinInteractionGraph extends React.Component<
     }
     return (
       <>
-        <div id={CHART_ID}></div>
+        <div id={GRAPH_ID}></div>
+        <ProteinProteinInteractionTable
```

Comment:
nit: you can actually call a component like 
`<ProteinProteinInteractionTable data={...} />` and not need to add a separate closing tag

Reply:
Thanks this is great syntax!